Mossharbor.AzureWorkArounds.ServiceBus/WebClientExtensions.cs

```diff
@@ -12,6 +12,7 @@ namespace Mossharbor.AzureWorkArounds.ServiceBus
     using System.Xml;
     using System.Xml.Serialization;
 
+    [Serializable]
```

Comment:
Why do you need to serialize and extension class?

Reply:
Indeed I don't, I made this just for caution, BizTalk is very sensible to objects serialization.